pkgs/servers/monitoring/uptime-kuma/package.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
is the package.json or package-lock.json really necessary for node2nix? Or just while updating and rebuilding node-packages.nix?

Reply:
I think it is necessary yes, it won't build if I remove them